raiden/ui/cli.py

```diff
@@ -689,6 +690,15 @@ def app(
         discovery,
     )
 
+    try:
+        raiden_app.start()
+    except filelock.Timeout:
+        print(
+            f'FATAL: Another Raiden instance already running for account {address_hex} on '
+            f'network id {network_id}',
```

Comment:
Maybe use `constants.ID_TO_NETWORKNAME` to translate the network id and `to_checksum_address` for the eth address to have a nicer message.